src/AppConstants.js

```diff
@@ -37,6 +38,30 @@ export const SEVERITY_MAP = {
   'medium-risk': 2,
   'low-risk': 1,
 };
+
+// Recommendations OverviewDashbarCards titles
+export const PATHWAYS = 'Pathways';
+export const INCIDENTS = 'Incidents';
+export const IMPORTANT_RECOMMENDATIONS = 'Important Recommendations';
+export const CRITICAL_RECOMMENDATIONS = 'Critical Recommendations';
+
+export const OverviewDashbarCardSupportedTitles = {
```

Comment:
Why do the titles need to be organised like this?

Reply:
Also, side note, we should only capitalise component names, this here is just an object. :)

Reply:
Changed the name of the object like you mentioned, thank you for noticing :) 

It used to be an Array, and then I checked if the supportedTitles Array `includes` the certain given title
For some reason it threw an error and I couldn't get it to work again (it was super weird because I haven't touched this code)
This is the same idea but implemented with an object (the object serves as a Set) if the title is found in the Set, the title is supported, and the component is rendered. 

In case we would want to add cards with different titles in the future, we can just add it to the Set (or remove it)

Reply:
After our conversation, I have removed this object entirely 👍🏼